pep-0526.txt

```diff
@@ -174,38 +178,23 @@ by a different type::
   a: int
   a: str  # Static type checker will warn about this.
 
-``__annotations__`` is writable, so this is permitted::
+.. _classvar:
```

Comment:
Here it looks like you lost a whole section of new stuff added previously about `__annotations__`.